generated/models/inputs.go

```diff
@@ -85,6 +85,7 @@ func (i GetBooksInput) Validate() error {
 // GetBookByIDInput holds the input parameters for a getBookByID operation.
 type GetBookByIDInput struct {
 	BookID        int64
+	AuthorID      *string
```

Comment:
i thought about making this `bson.ObjectId`, but held off because it wouldn't be that in `book.go`. What do you think?